landing-strip/core/src/main/java/com/novoda/landingstrip/State.java

```diff
@@ -49,4 +63,11 @@ public void invalidateFastForwardPosition() {
     public boolean fastForwardPositionIsValid() {
         return getFastForwardPosition() != -1;
     }
+
+    public Coordinates getIndicatorPosition(TabsContainer tabsContainer) {
```

Comment:
would it make more sense to have this in IndicatorCoordinatesCalculator, where TabsContainer is a field, and State is passed in?